tests/scripts/expect/cli-tcat.exp

```diff
@@ -58,6 +63,26 @@ send "ping 512\n"
 expect_line "\tTYPE:\tRESPONSE_W_PAYLOAD"
 expect_line "\tLEN:\t512"
 
+send "network_name\n"
+expect_line "\tTYPE:\tRESPONSE_W_PAYLOAD"
+expect_line "\tLEN:\t15"
+expect_line "\tVALUE:\t0x4f70656e5468726561642d63363465"
+
+send "device_id\n"
+expect_line "\tTYPE:\tRESPONSE_W_PAYLOAD"
+expect_line "\tLEN:\t8"
+expect_line "\tVALUE:\t0x18b4300000000001"
```

Comment:
If no particular device ID is configured by the vendor,  and I assume by default the OpenThread build doesn't configure such ID (only if vendor adds some build-time flag I suppose?) ; then I would expect the code falls back to sending the EUI-64.  What value would we then expect here? Is this the EUI-64 here of the simulated node in the expect tests ?

Reply:
This is exactly why I had a concern here: https://github.com/openthread/openthread/pull/10526#discussion_r1686290981

Reply:
But the concern was about Extended Address, and my expectation was on the EUI-64. (Now I'm confused.) Maybe we can just go ahead with whatever ID and I can propose a PR in a few weeks time, if needed, to fix it.

Reply:
This EUI-64 is constructed based on node ID of the simulated node [here](https://github.com/openthread/openthread/blob/main/examples/platforms/simulation/radio.c). 

The new ID was already introduced. please refer to this https://github.com/openthread/openthread/pull/10526#discussion_r1700680694. More or less this should be fine now.